utils/build_env.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""build environment used by shell scripts
+"""
+
+# set path
+import sys
+from os.path import realpath, dirname, join, sep
+repo_root = realpath(dirname(realpath(__file__)) + sep + '..')
+sys.path.insert(0, repo_root)
+
+from searx import brand
```

Comment:
Here `searx.brand` is based on `searx.settings`.

If `SEARX_SETTINGS_PATH` is set, then `utils/brand.env` won't have the expected values from `searx/settings.yml`.

see commit https://github.com/dalf/searx/commit/6faaf5e976c23bd03abf406990b1501b5463bdaa

Reply:
Hm, I had to think twice about .. but under the assumption that a brand is always a fork, you are right.  I will cherry pick / thanks!

Reply:
Actually I had the issue because `make test.robot` was not working, [without log](https://github.com/searx/searx/blob/dbfd5567aaa06a5f9792bb0839af74877ed5258c/searx/testing.py#L49-L56), I started searx with `SEARX_SETTINGS_PATH=$(pwd)/searx/testing.yml`, then `make test` started to fail.

So in the end, without the issue in `_brand_namespace.get_val`, I wouldn't have seen it.